Mathlib/RingTheory/Nilpotent/Exp.lean

```diff
@@ -165,3 +167,87 @@ theorem exp_of_nilpotent_is_unit {a : A} (h : IsNilpotent a) : IsUnit (exp a) :=
   rw [← exp_add_of_commute h₁.symm h₂ h, neg_add_cancel a, exp_zero_eq_one]
 
 end IsNilpotent
+
+namespace LinearMap
+
+variable {R M N : Type*} [CommRing R]
+  [AddCommGroup M] [Module R M] [AddCommGroup N] [Module R N]
+
+variable (M) in
+theorem isNilpotent_lTensor_of_isNilpotent (f : Module.End R N) (hf : IsNilpotent f) :
```

Comment:
Could also call this `IsNilpotent.linearMapLTensor` if you prefer.

Reply:
Done.